vm/vm.go

```diff
@@ -201,8 +201,9 @@ func (vm *VM) initConstants() {
 		vm.initChannelClass(),
 		vm.initGoClass(),
 		vm.initFileClass(),
+		vm.initRegexpClass(),	
```

Comment:
Commit f5e3e20b introduces, likely accidentally, a trailing space on this line, and a misaligned brace two lines after - I think they should be reverted.

Reply:
Got it